examples/inverse/plot_dics_beamformer.py

```diff
@@ -55,13 +55,15 @@
 # Read forward operator
 forward = mne.read_forward_solution(fname_fwd)
 
-# Computing the data and noise cross-spectral density matrices
-# The time-frequency window was chosen on the basis of spectrograms from
-# example time_frequency/plot_time_frequency.py
-data_csd = csd_epochs(epochs, mode='multitaper', tmin=0.04, tmax=0.15,
-                      fmin=6, fmax=10)
-noise_csd = csd_epochs(epochs, mode='multitaper', tmin=-0.11, tmax=0.0,
-                       fmin=6, fmax=10)
+# Computing the data and noise cross-spectral density matrices using
+# multitapers. The time-frequency window was chosen on the basis of
+# spectrograms from example time_frequency/plot_time_frequency.py
+data_csd = csd_multitaper(epochs, tmin=0.04, tmax=0.15, fmin=6, fmax=10)
+noise_csd = csd_multitaper(epochs, tmin=-0.11, tmax=0.0, fmin=6, fmax=10)
+
+# Average the CSDs across the frequencies
+data_csd = data_csd.mean()
+noise_csd = noise_csd.mean()
```

Comment:
are the CSD plotting functions anywhere in the doc?

Reply:
In the API, but not as an example yet. I'm planning to add a small example comparing the three methods side by side. That would also show off the plotting function.